frappe/public/js/frappe/form/footer/attachments.js

```diff
@@ -209,11 +209,13 @@ frappe.ui.form.Attachments = Class.extend({
 frappe.ui.get_upload_dialog = function(opts){
 	dialog = new frappe.ui.Dialog({
 	    title: __('Upload Attachment'),
-		no_focus: true,
+			no_focus: true,
 	    fields: [
-			{fieldtype: "Section Break"},
+			{"fieldtype": "Section Break"},
 			{"fieldtype": "Link" , "fieldname": "file" , "label": __("Select uploaded file"), "options": "File"},
-	    ],
+			{"fieldtype": "Section Break", "label": __("Gsuite Document")},
```

Comment:
these fields should be hidden by default unless gsuite is integrated. You can add a property in `boot.py` that you can access as `frappe.boot`